packages/expo-image/ios/EXImage.podspec

```diff
@@ -1,6 +1,6 @@
 require 'json'
 
-package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
```

Comment:
Are you sure `expo-image` will be autolinked by RN when you move the podspec? As far as I remember, the CLI looks for them only in the root dir if not specified in `react-native.config.js`.

Reply:
good catch 👍  i didn't notice expo-image using cli linking. i'll move the podspec back.